features/delete_for_review.feature

```diff
@@ -0,0 +1,360 @@
+@isolated
+Feature: cli-kintone delete command
+
+  Scenario: CliKintoneTest-126 Should return the error message when the user has no privilege to delete records.
```

Comment:
It seems that token permissions are used to verify (not user privilege).
How about updating the scenario like this?
`when the user has no privilege to delete records`
↓
`with an API Token without Delete permission`